compose/ui/ui-uikit/src/uikitMain/objc/CMPUIKitUtils/CMPUIKitUtils/CMPInteropWrappingView.h

```diff
@@ -0,0 +1,19 @@
+//
+//  CMPInteropWrappingView.h
+//  CMPUIKitUtils
+//
+//  Created by Ilia.Semenov on 05/04/2024.
+//
+
+#import <UIKit/UIKit.h>
+#import "CMPMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CMPInteropWrappingView : UIView
```

Comment:
LGTM, One thing to think about:
Creating new subclass of `UIView` / `UIAccessibilityElement` for every new use-case (which is now) VS create one subclass of UIView, one of UIAccessibilityElement with all default implementations of necessary methods from Accessibility protocols.

Reply:
The whole point of this is to make the UIAccessibility methods be visible to Kotlin in the first place. Cinterop doesn't see some of them because they are declared in a category and struggles to do overriding because of duality of property and selectors in ObjC which is not the case for fields and methods in Kotlin. Keeping it in ObjC and avoiding presence of subclasses in Kotlin altogether would require adding a lot of code to ObjC to map the values and types from compose core. Thus I find the current solution the most optimal trade-off.